package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@netzstrategen/twig-drupal-fractal-adapter",
-  "version": "2.2.1",
+  "version": "2.3.0",
```

Comment:
This appears to be a straightforward bugfix to me?
```suggestion
  "version": "2.2.2",
```

Reply:
Makes total sense to me. I'm not sure anymore why this was bumped minor instead.